### PR TITLE
Disable detectLang on translateDoc interface, enable otherwise

### LINKED ISSUE #174: Disable 'detectLanguage' when on 'docTranslation' interface?
When on `docTranslation` interface, on clicking `detect`, it takes up the contents of `originalText` and does the detect. But, one would expect the functionality to detect the language of the contents inside the uploaded doc thereby making it misleading. **Does the `detectLanguage` functionality work for docs too?**
(I could only see the functionality for `translateText` interface).

If no, would it make sense to disable the `detect` button while on `docTranslation` interface and then enable it again while switching to `translateText` interface?

### PR DESCRIPTION
**Pull request by share-with-me**
Fix #174

### CODE REVIEW

File: assets/js/translator.js [hunk 1]
```diff
@@ -223,6 +223,8 @@ if(modeEnabled('translation')) {
                 $('#fileInput').show();
                 $('div#fileName').hide();
                 $('div#docTranslation').fadeIn('fast');
+                $('#detect').prop('disabled', true);
+                $('#srcLangSelect').children('option[value=' + 'detect' + ']').attr('disabled', true);
```
Comment (sushain97) [inline assets/js/translator.js]: Why are you using concatenation here?

Comment (sushain97) [inline assets/js/translator.js]: I would suggest actually combining these into a "single" selector with a comma.

Comment (share-with-me) [inline assets/js/translator.js]: I followed the style as used across the code, but yeah, we are looking only for a constant `string`, I'd change it now!

Comment (share-with-me) [inline assets/js/translator.js]: Awesome, makes sense

Comment (share-with-me) [inline assets/js/translator.js]: Um @sushain97 , are you suggesting something like following? (using a jquery inside another selector)?  The `option` value does not have an ID. Thus I'll need to select `detect` from the `select` dropdown and disable the option. Referred [this](https://stackoverflow.com/questions/10525651/jquery-selector-element-inside-other)

`$('#detect', $('#srcLangSelect').children('option[value=detect]')).attr('disabled', true);`

Comment (sushain97) [inline assets/js/translator.js]: Why are you using children?

**Comment by share-with-me** [comment]
Combined in selector. Sorry for poor glitches on this one :) . I was using the `select option` through jquery API for first time. Have read the API usages now.

Thanks!

**Pull request merged**